File: src/main/java/com/aagu/blog/controller/LogController.java
package com.aagu.blog.controller;

import com.aagu.blog.aspect.Auth;
import com.aagu.blog.service.LogService;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestParam;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/logs")
class LogController {

    private final LogService logService;

    public LogController(LogService logService) {
        this.logService = logService;
    }

    @GetMapping("")
    @Auth
    public Object getLog(@RequestParam(value = "page", defaultValue = "1") Integer page,
                         @RequestParam(value = "limit", defaultValue = "10")Integer limit) {
        return logService.getLogs(page, limit);
    }
}

File: src/main/java/com/aagu/blog/controller/LogController.kt
package com.aagu.blog.controller

import com.aagu.blog.service.LogService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RequestParam
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/logs")
class LogController {
    @Autowired
    private lateinit var logService: LogService

    @GetMapping("")
    fun getLog(@RequestParam("page", defaultValue = "1") page: Int,
               @RequestParam("limit", defaultValue = "10") limit: Int): Any {
        return logService.getLogs(page, limit)
    }
}